api/lib/schemas/menu.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+    "type": "object",
+    "properties": {
+        "_id": {
+            "type": "string"
+        },
+        "_rev": {
+            "type": "string"
+        },
+        "restaurant": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string"
+                },
+                "address": {
+                    "type": "string"
+                },
+                "city": {
+                    "type": "string"
+                }
+            },
+            "required": [ "name", "address", "city" ],
+            "additionalProperties": false
+        },
+        "file": {
+            "type": "string"
+        },
+        "ocr": {
+            "type": "array",
+            "items": [
+                { "type": "string" }
+            ]
+        }
+    },
+    "required": [ "restaurant"],
```

Comment:
Just wondering, do you think the file is also required?

Reply:
Sure thing! I forgot to put it in the required field! Thanks for point this out.